backend/sftp/ssh_external.go

```diff
@@ -175,16 +178,19 @@ func (s *sshSessionExternal) exited() bool {
 
 // Wait for the command to exit
 func (s *sshSessionExternal) Wait() error {
-	if s.exited() {
-		return nil
-	}
-	err := s.cmd.Wait()
-	if err == nil {
-		fs.Debugf(s.f, "ssh external: command exited OK")
-	} else {
-		fs.Debugf(s.f, "ssh external: command exited with error: %v", err)
-	}
-	return err
+	// Use sync.Once to ensure we only wait for the process once
+	// This prevents zombie processes that occur when Wait() is called multiple times
```

Comment:
This comment is wrong - zombies do not ocurr when Wait() is called multiple times.